### PR TITLE
implemented hairstyle database tables

### PR DESCRIPTION
**Pull request by YochevedG**
I did the work, please review and let me know if anything needs to be changed

### CODE REVIEW

File: 1 table.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        when AppointmentType = 'HairCut'
+        then 20
+        when AppointmentType = 'Keratin'
+        then 400
+        when AppointmentType = 'HairStyle'
+        then 65
+        when AppointmentType = 'WigWash'
+        then 50
+        when AppointmentType = 'WigStyle'
+        then 100
```
Comment (Devorag) [inline 1 table.sql:38]: no need to repeat 'appointment type' so many times. you can easily write as case appointment type and then just leave it out.

File: 3 reports.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+--Reports:
+--1) A list of all clients who's charge is less than $125
+select * from HairStyle h
+where h.Fee < 125
+--2) The average profit earned in the 1990's
+select AvgPofit = avg(h.fee)
+from HairStyle h
+where year(h.AppointmentDate) between 1900 and 1999
+--3) The number of patients who's last name has an 'e' or their phone number has a '6' or '7'
+select count(*)
```
Comment (Devorag) [inline 3 reports.sql:11]: please insert column titles

**Review by Devorag — CHANGES_REQUESTED** [review]

File: 3 reports.sql (hunk 2 above)
Comment (Devorag) [reply 3 reports.sql:11]: I reviewed the work, excellent job!!! Please see comments. You are amazing!!!!!!!!!!!

**Comment by Devorag** [comment]
I reviewed the work, excellent job!!! Please see comments. You are amazing!!!!!!!!!!!

File: 1 table.sql (hunk 1 above)
Comment (YochevedG) [reply 1 table.sql:38]: ty, done!

File: 3 reports.sql (hunk 2 above)
Comment (YochevedG) [reply 3 reports.sql:11]: got it, fixed!

**Comment by YochevedG** [comment]
thanks for the comments!
I fixed what you requested, please review

File: 1 table.sql
Comment (Devorag) [inline 1 table.sql:1]: Great!

File: 3 reports.sql
Comment (Devorag) [inline 3 reports.sql:1]: Great!

**Comment by Devorag** [comment]
Fantastic! keep up the great work!

**Review by Devorag — COMMENTED** [review]
Thank you!!!!!!!!!!!!!!!

**Pull request closed without merging**